repository-hbase/src/main/java/hbase/load/CSVDataKVMapper.java

```diff
@@ -151,7 +153,7 @@ private void writeRow(Context context, String rowKeyStr, String[] fields) throws
                 try {
                     put.add(new KeyValue(rowKey.get(), columnFamily, columnNames[i], fields[i].getBytes()));
                 } catch (Exception e) {
-                    LOG.error("Cannot add line '{}'", fields[0], e);
+                    LOG.error("ColumnName: '{}' \n.Cannot add line '{}' with header '{}'", columnNames, fields[0], columnNames[i], e);
```

Comment:
Believe errors need to be added to a central counter in order to report back at the end when results are collated post distribution. Ask Ranjit

Reply:
I have noticed the BAD_HBASE_RECORDS counter is incremented if there is any failure while writing to HFile. So I think its ok if we don't increase the counter here while processing all the fields of a record.